spec/lib/triannon-client/triannon-client_spec.rb

```diff
@@ -91,86 +177,127 @@ def check_graph_has_statements(graph)
       end
     end
   end
 
   describe "#get_iiif_annotation" do
     # the mime type is fixed as 'ld+json' for this method
-    it 'should get an open annotation by ID, using a IIIF profile' do
-      graph = @tc.get_iiif_annotation(@anno[:id])
+    it 'requests an open annotation by ID, using a IIIF profile' do
+      anno = create_annotation
+      id = anno[:id]
+      tc = TriannonClient::TriannonClient.new
+      graph = tc.get_iiif_annotation(id)
       check_graph_has_statements(graph)
+      delete_annotation(id)
       #TODO check that @tc client sends a request with the right profile header
     end
   end
 
   describe "#get_oa_annotation" do
     # the mime type is fixed as 'ld+json' for this method
-    it 'should get an open annotation by ID, using an OA profile' do
-      graph = @tc.get_oa_annotation(@anno[:id])
+    it 'requests an open annotation by ID, using an OA profile' do
+      anno = create_annotation
+      id = anno[:id]
+      tc = TriannonClient::TriannonClient.new
+      graph = tc.get_oa_annotation(id)
       check_graph_has_statements(graph)
+      delete_annotation(id)
       #TODO check that @tc client sends a request with the right profile header
     end
   end
 
   describe "#post_annotation" do
-    it 'should POST an open annotation' do
-      expect{@tc.post_annotation(@oa_jsonld)}.not_to raise_error
+    it 'does not raise an error when submitting a valid open annotation' do
+      tc = TriannonClient::TriannonClient.new
+      response = nil
+      expect do
+        response = tc.post_annotation(@oa_jsonld)
+      end.not_to raise_error
+      # Check the POST by deleting the annotation.
+      # If the POST was successful, the DELETE should work too.
+      graph = tc.response2graph(response)
+      uri = tc.annotation_uri(graph)
+      id = tc.annotation_id(uri)
+      expect(tc.delete_annotation(id)).to be true
+      # TODO: stub and mock this OR test the response is an Open Anno?
     end
     it 'returns a RestClient::Response object' do
       # The response behaves primarily as a String, so it can not be tested
       # as an instance of RestClient::Response, but it can be tested to respond
-      # to RestClient methods, i.e.
-      r = @anno[:response]
-      expect(r.respond_to? 'code').to be_truthy
-      expect(r.respond_to? 'body').to be_truthy
-      expect(r.respond_to? 'headers').to be_truthy
+      # to RestClient methods.
+      anno = create_annotation
+      r = anno[:response]
+      expect(r).to respond_to(:code)
+      expect(r).to respond_to(:body)
+      expect(r).to respond_to(:headers)
+      delete_annotation(anno[:id])
     end
   end
 
   describe '#response2graph' do
-    it 'turns RestClient::Response into RDF::Graph' do
-      expect(@anno[:graph]).to be_instance_of RDF::Graph
+    it 'returns an RDF::Graph' do
+      anno = create_annotation
+      expect(anno[:graph]).to be_instance_of RDF::Graph
+      delete_annotation(anno[:id])
     end
   end
```

Comment:
but i don't know that from looking at the test.  can you make the test call response2graph directly, or, minimally, add a comment to that effect?